boot/bootdata/hivecls.inf

```diff
@@ -220,7 +220,9 @@ HKCR,"chm.file\shell\open\command","",0x00020000,"%SystemRoot%\hh.exe %1"
 HKCR,".hta","",0x00000000,"htafile"
 HKCR,"htafile","",0x00000000,"HTML Application"
 HKCR,"htafile\DefaultIcon","",0x00020000,"%SystemRoot%\system32\mshta.exe,-1"
-HKCR,"htafile\shell\open\command","",0x00020000,"%SystemRoot%\system32\mshta.exe ""%1"" %*"
+; HKCR,"htafile\shell\open\command","",0x00020000,"%SystemRoot%\system32\mshta.exe ""%1"" %*"
+HKCR,"htafile\shell\edit\command","",0x00020000,"%SystemRoot%\system32\notepad.exe %1"
+HKCR,"htafile\shell\open\command","",0x00020000,"""%ProgramFiles%\Internet Explorer\iexplore.exe"" %1"
```

Comment:
Don't we need also to (double)quote the %1 ?

Reply:
I had copied the string from internet explorer and changed the file type. It worked for me in my limited testing.

Reply:
Depends how they handle paths.
If they concatenate the arguments to form one path, it works fine without quotes

Reply:
Should I double quote it?

Reply:
One way to find out: put a .hta file in a path with spaces and see if your commands still work.

Reply:
I placed an hta file in a foler that had a space in the name and it worked for me.